### DISCUSSION TITLE
WooCommerce Endpoint Styling

### DISCUSSION THREAD

**Discussion by DudeDude95** [elementor/elementor#22922] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor provides the functionality to style the checkout page which is awesome!

However, not all customers take the same route to the checkout by adding products to cart and navigating to the checkout themselves ... Many stores will manually create an order in the WooCommerce Dashboard on behalf of the customer and then send the customer the "Customer Payment Page" to simply enter their payment details:
![image](https://github.com/elementor/elementor/assets/123473359/a407f62a-4ecb-47c6-b04a-f93e496bb0ac)

Although the "Customer Payment Page" uses the checkout template, it users the `order-pay` endpoint.

I have customised my checkout template to the max but there are no options to customise the `order-pay` endpoint anywhere.

This means when I send the customer the "Customer Payment Page" they see this default looking page:
![image](https://github.com/elementor/elementor/assets/123473359/f95fc071-6147-4d13-9709-d07dcbfa2a24)

### Describe the solution you'd like

I would like the ability to customise the WooCommerce order-pay endpoint, the "Customer Payment Page", with Elementor.

### Describe alternatives you've considered

Of course I can use Custom CSS but if feels like the functionality is already halfway there if the main checkout template can be customised.

**Comment by User222009** [comment]
> ### Prerequisites
> 
>     * [x]  I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
> 
>     * [x]  The feature is still missing in the latest stable version of Elementor ( Elementor Pro )
> 
> 
> ### What problem is your feature request going to solve? Please describe.
> 
> Elementor provides the functionality to style the checkout page which is awesome!
> 
> However, not all customers take the same route to the checkout by adding products to cart and navigating to the checkout themselves ... Many stores will manually create an order in the WooCommerce Dashboard on behalf of the customer and then send the customer the "Customer Payment Page" to simply enter their payment details: ![image](https://private-user-images.githubusercontent.com/123473359/249425648-a407f62a-4ecb-47c6-b04a-f93e496bb0ac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNjE5MzUsIm5iZiI6MTcxODI2MTYzNSwicGF0aCI6Ii8xMjM0NzMzNTkvMjQ5NDI1NjQ4LWE0MDdmNjJhLTRlY2ItNDdjNi1iMDRhLWY5M2U0OTZiYjBhYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxM1QwNjUzNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MmYwZjgxY2YyYzgzMWY3ZTQzNmNmNmQ5YWFjNWIyODM5MGU0M2M2NDI3MzA3NTFkZGYwMTY4ODlmOTE1ZTIzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9._mkbMnqdiX_jrxPnOnpC_G3EFenZAUDgbaKhKaQOY3o)
> 
> Although the "Customer Payment Page" uses the checkout template, it users the `order-pay` endpoint.
> 
> I have customised my checkout template to the max but there are no options to customise the `order-pay` endpoint anywhere.
> 
> This means when I send the customer the "Customer Payment Page" they see this default looking page: ![image](https://private-user-images.githubusercontent.com/123473359/249431802-f95fc071-6147-4d13-9709-d07dcbfa2a24.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgyNjE5MzUsIm5iZiI6MTcxODI2MTYzNSwicGF0aCI6Ii8xMjM0NzMzNTkvMjQ5NDMxODAyLWY5NWZjMDcxLTYxNDctNGQxMy05NzA5LWQwN2RjYmZhMmEyNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYxM1QwNjUzNTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02YjVhYWI2ZDMxZTA0ZDhiNWEwMDU1OTZiMTk5MmJkNTVhMTg1ZDUwMjFkNGRmMTc5ZWNkYzEyYTgwNmQwZjI0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.WgYz3-Q1Z6DT1WgeQSvs1iOJI_6Q7Y0AFXoyjdCaCMI)
> ### Describe the solution you'd like
> 
> I would like the ability to customise the WooCommerce order-pay endpoint, the "Customer Payment Page", with Elementor.
> ### Describe alternatives you've considered
> 
> Of course I can use Custom CSS but if feels like the functionality is already halfway there if the main checkout template can be customised.
> ### Additional context
> 
> _No response_

Hi, have you found a solution to this yet? From what I see only certain text colours edited on the Checkout Page - Payment section apply to the order-pay endpoint. I tried some css to change at least the background for the Payment section and it won't work for me.
